app/app.py

```diff
@@ -33,6 +33,24 @@
 
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
+# Prompt Templates
+SIMPLE_TEMPLATE = "Context: {context}\n\nQuestion: {question}\n\n"
+
+COMPLEX_TEMPLATE = """Use the following pieces of context to answer the question at the end.
+If you don't know the answer, just say that you don't know, don't try to make up an answer.
```

Comment:
Instead of just asking the model these, maybe it will be better to specify them as rules and instruction such as:

```
The conversation should abide by the following rules and instructions:
1. If you don't know the answer, just say that you don't know, don't try to make up an answer.
2. ...
```
I have seen the use of this prompt in [NHS-LLM](https://github.com/CogStack/OpenGPT/blob/main/data/prompts.json) which seems to generate good result.

Reply:
Yeah, this will be part of experimentation on different prompts. I have used the same prompt as default that was used in colab notebook.